lang/semgrep-grammars/src/semgrep-swift/test/corpus/semgrep.txt

```diff
@@ -23,3 +23,70 @@ Unbounded range operator
   (additive_expression
     (tuple_expression (fully_open_range))
     (integer_literal)))
+
+================================================================================
+Param Ellipsis
```

Comment:
Those are calls arguments, not parameters :)

Reply:
:facepalm: